beacon-chain/node/node.go

```diff
@@ -474,6 +506,12 @@ func (b *BeaconNode) startStateGen() error {
 	r := bytesutil.ToBytes32(cp.Root)
 	// Consider edge case where finalized root are zeros instead of genesis root hash.
 	if r == params.BeaconConfig().ZeroHash {
+		if sg.MinimumSlot() > 0 {
```

Comment:
build fails

Reply:
Yeah, this was made private in one of the other PRs and broke after rebase - I'm working on axing all this minimum slot stuff, because we're going to just require genesis state to be provided.

Reply:
obsoleted by other changes